default_lcf.xml

```diff
@@ -119,5 +119,11 @@
         <!-- <Logger name="jmri.jmrit.display.layoutEditor.LayoutBlock" level="DEBUG"/> -->
         <!-- <Logger name="jmri.jmrit.operations" level="DEBUG"/> -->
 
+        <!-- //Mey -->
+        <Logger name="jmri.jmrix.bidib" level="INFO"/>
+        <Logger name="org.bidib.wizard" level="WARN"/>
+        <Logger name="org.bidib.jbidibc" level="WARN"/>
+        <Logger name="myScripts" level="INFO"/>
+
```

Comment:
Is this change intended?

Reply:
No, thats private. I thought changes to this file would not be merged. May be removed.

Reply:
It's best if you can remove these lines and commit it to this branch.

Reply:
> No, thats private. I thought changes to this file would not be merged. May be removed.

@bobjacobsen
Is there a way for him to add these lines privately without commiting it to JMRI?

Reply:
Hi Folks, see https://www.jmri.org/help/en/html/apps/Debug.shtml#logging_configuration

A "default_lcf.xml" file in the User Settings directory will take precedence over the default_lcf.xml file in the main program directory.

Reply:
Oh, I didn't know that we can have it in die User Settings directory! Thanks for that advice. I have reset the file to the original file from master and pushed that to the emey-bidib-update-jbidibc branch.
BTW: It could be a good idea if this file is looked up in the profile directory in the first place so we could have different settings in the "production" profile and e.g. a test profile with a simulation.